core/schema/container.go

```diff
@@ -184,26 +180,26 @@ func (s *containerSchema) Install() {
 			ArgDoc("name", `The name of the label to remove (e.g., "org.opencontainers.artifact.created").`),
 
 		dagql.Func("entrypoint", s.entrypoint).
-			Doc(`Retrieves entrypoint to be prepended to the arguments of all commands.`),
+			Doc(`Return the container's OCI entrypoint.`),
 
 		dagql.Func("withEntrypoint", s.withEntrypoint).
-			Doc(`Retrieves this container but with a different command entrypoint.`).
-			ArgDoc("args", `Entrypoint to use for future executions (e.g., ["go", "run"]).`).
-			ArgDoc("keepDefaultArgs", `Don't remove the default arguments when setting the entrypoint.`),
+			Doc(`Set an OCI-style entrypoint. It will be included in the container's OCI configuration. Note, \"withExec\" ignores the entrypoint by default.`).
```

Comment:
```suggestion
			Doc(`Set an OCI-style entrypoint. It will be included in the container's OCI configuration. Note, "withExec" ignores the entrypoint by default.`).
```

The escapes aren't needed, this is a raw string (as is, this will actually insert the `\` into the generated docs).